cmd/skywire-cli/commands/config/gen.go

```diff
@@ -121,24 +123,25 @@ var genConfigCmd = &cobra.Command{
 			logger.Fatal("Use of mutually exclusive flags: -w --hide and -n --stdout")
 		}
 		if dmsgHTTP {
+			dmsgHTTPPath := skyenv.DMSGHTTPPath
 			if pkgEnv {
```

Comment:
@mrpalide @0pcom do we need to use a different dmsghttp config path for package?

Reply:
`/opt/skywire/dmsghttp-config.json` for the linux packages
something different for windows & mac I'm sure

Reply:
Main problem here is when trying to run `skywire-cli config gen ...` in a path different from location of files.
As Moses said for linux packages is `/opt/skywire/dmsghttp-config.json`.
For windows be good if we disable choosing install path by users, and just use `C:/Program Files/Skywire/dmsghttp-config.json`. Also same for Mac but I don't know about it.

Reply:
I can add the windows and mac paths. Can someone just confirm the Mac path please?

Reply:
Based on Mac Installer scripts, Mac path is correct.
And about the second question, we use another output for config generated, `"%HOMEPATH%\skywire-config.json"`.

Reply:
So the `skywire-config.json` and `dmsghttp-config.json` are in a different location in the windows installer?

Reply:
Yes